src/v2/view-builder/views/password/ReEnrollAuthenticatorWarningPasswordView.js

```diff
@@ -5,17 +5,21 @@ import EnrollAuthenticatorPasswordView from './EnrollAuthenticatorPasswordView';
 const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
   className: 'password-authenticator',
   title () {
-    const daysToExpiry = this.options.appState.get('currentAuthenticator').settings.daysToExpiry;
-    return `${loc('password.expiring.title', 'login', [ daysToExpiry ])}`;
+    const passwordPolicy = this.getPasswordPolicy() || {};
```

Comment:
on a second thought it makes sense to rename the function to `getPasswordPolicySettings`